bin/helm-deploy

```diff
@@ -7,6 +7,16 @@ set -eo pipefail
 
 HELM_DEFAULT_TIMEOUT=300
 
+format_multiple_values_files() {
+  values_files="$1"
+  formatted_files=""
+  echo "${values_files}" | sed -n 1'p' | tr ',' '\n' | ( while read file_name; do
+    file="./deploy/${file_name}.values.yml,"
```

Comment:
This will leave a trailing `,` which throws an error for me with helm 2.9.1
```
helm install stable/redis -f file.yml,file2.yml,
Error: open : no such file or directory
```

Reply:
i have a fix for that i will add.